gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -3207,7 +3209,7 @@ subroutine GFS_physics_driver                                 &
       endif   ! end if_ldiag3d/lgocart/cplchm
 
       if (Model%lgocart .or. Model%cplchm) then
-        Coupling%dqdti(1:im,:) = 0._kind_phys
+        Coupling%dqdti(1:im,:) = zero
```

Comment:
Why you changed "0._kind_phys" to "zero"? If they are the same, I think it is better to keep the original "0._kind_phys"

Reply:
I think this is version different, in my version , it used zero as default, nothing change. Better to keep your original version.